packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall.html

```diff
@@ -156,6 +156,7 @@
                 <tr data-ng-if="IpFirewallCtrl.isAddingRule">
                     <td>
                         <oui-select
+                            class="position-initial"
```

Comment:
**question:** Where are the classes `position-initial` & `overflow-visible` defined? They are not fro bootstrap utilities as well.